electron-app/src/views/settings/Accounting.vue

```diff
@@ -101,7 +118,7 @@
                       </template>
                     </v-badge>
                   </div>
-                </v-select>
+                </v-select> -->
```

Comment:
I am guessing the commented code is in the draft only right?

Reply:
Yep.